src/TLuaInterpreter.cpp

```diff
@@ -12899,3 +12901,63 @@ Host& getHostFromLua(lua_State* L)
     assert(h);
     return *h;
 }
+
+int TLuaInterpreter::getColumnCount( lua_State * L )
+{
+    string luaWindowName = "";
+
+    if ( lua_gettop( L ) && ! lua_isstring( L, 1 ) ) {
+        lua_pushstring( L, "getColumnCount: wrong argument type" );
```

Comment:
We're looking to migrate away from this old error style which is not very descriptive and thus helpful - to a newer one where we say "blah expected, got <this>". For example, check out https://github.com/Mudlet/Mudlet/blob/development/src/TLuaInterpreter.cpp#L2403. Could you update the function to use that?

You'll also notice the function allows non-Latin window names 😉

Reply:
Thanks for the input.

I've updated and rebased the PR. Hope that it's better now.